src/frontend/css/general.css

```diff
@@ -44,15 +44,82 @@ div.app-container {
   height: max(100vh, 720px);
   width: max(100vw, 1080px);
   display: grid;
-  grid-template-areas: "left right";
+  grid-template-areas: "game board panel";
   grid-template-rows: auto;
-  grid-template-columns: 75% 25%;
+  grid-template-columns: 25% 50% 25%;
   align-items: center;
   justify-items: center;
 }
 
+/* Game Panel */
+
+div.game-panel-container {
+  grid-area: game;
+  height: 100%;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+/*   border: red 1px solid; */
```

Comment:
I've left these `border` comments in for now as that styling is useful for figuring out visual issues.

Reply:
Good idea. Maybe we should have our own thing in `%chess` like `XX` in Hoon code, as a signal that, "this is debug code and should be commented out unless you're actively looking for issues; please do not remove it". Maybe `DD`?